### ISSUE TITLE
Interpolate crossing between points straddling zone edge.

### ISSUE THREAD

**Issue by philderbeast** [BlockScope/CIVL-GAP#7]
Surely the crossing interpolation would be between points `j` and `j+1` straddling the cylinder zone's edge and not between `j+1` and itself, when `j+1` is either wholly inside or wholly outside the zone. This typo appears twice in the original docs and I transcribed each one as-is

https://github.com/BlockScope/CIVL-GAP/blob/e26334b23531dd8b314d1c32c5304fc4aef2121a/gap.tex#L835

https://github.com/BlockScope/CIVL-GAP/blob/e26334b23531dd8b314d1c32c5304fc4aef2121a/gap.tex#L867

**Comment by joergewald** [comment]
Correct, this is a typo, in both instances. should be interpolateTime(trackpoint_{j}, trackpoint_{j + 1})
